.github/workflows/release.yml

```
@@ -632,3 +632,14 @@ jobs:
                   --arg channel "#xrpl-js" \
                   --arg text "$MESSAGE" \
                   '{channel: $channel, text: $text}')"
+
+  generate-documentation:
+    name: Generate and Publish documentation for ${{ needs.get_version.outputs.package_version }}
+    uses: ./.github/workflows/generate-documentation.yml
+    needs: [get_version, release]
+    with:
+      git_ref: ${{ github.event.inputs.release_branch }}
```

actionlint (1.7.8)

property "release_branch" is not defined in object type {npmjs_dist_tag: string; package_name: string}

(expression)